frontend/src/components/utils/ButtonRadioGroup.tsx

```diff
@@ -47,6 +47,7 @@ export type ButtonRadioGroupProps = {
   isDisabled?: boolean;
   options: string[];
   onChange: (newState: string) => void;
+  language?: string;
```

Comment:
Since we could use ButtonRadioGroup for other things in the future, mind finding a more agnostic name for the `language` prop? Thoughts on `value` or `dependentValue`?